ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core/BlockDataGenerator.java

```diff
@@ -399,7 +399,6 @@ private Transaction eip1559Transaction(final Bytes payload, final Address to) {
     return Transaction.builder()
```

Comment:
We definitely need to fix this to randomly include/not include access lists. Probably makes it more worth it to use the generator in the `TransactionEIP1559Test` because then we would immediately notice that this can't include access lists.